src/objectTranslations/Account-ca.objectTranslation

```diff
@@ -1,52 +1,52 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomObjectTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
-    <fields>
-        <help>L&apos;adreça actual que omplen els camps de facturació del compte.</help>
-        <label>Adreça actual</label>
-        <name>Current_Address__c</name>
-        <relationshipLabel>Comptes (adreça actual)</relationshipLabel>
-    </fields>
-    <fields>
-        <label>Primary Contact</label>
-        <name>Primary_Contact__c</name>
-        <relationshipLabel>Compte</relationshipLabel>
-    </fields>
-    <layouts>
-        <layout>HEDA Household Layout</layout>
-        <sections>
-            <label>Adreça</label>
-            <section>Address</section>
-        </sections>
-        <sections>
-            <label>Enllaços personalitzats</label>
-            <section>Custom Links</section>
-        </sections>
-        <sections>
-            <label>Descripció</label>
-            <section>Description</section>
-        </sections>
-        <sections>
-            <label>Camps</label>
-            <section>Fields</section>
-        </sections>
-    </layouts>
-    <layouts>
-        <layout>HEDA Organization Layout</layout>
-        <sections>
-            <label>Adreça</label>
-            <section>Address</section>
-        </sections>
-        <sections>
-            <label>Enllaços personalitzats</label>
-            <section>Custom Links</section>
-        </sections>
-        <sections>
-            <label>Descripció</label>
-            <section>Description</section>
-        </sections>
-        <sections>
-            <label>Camps</label>
-            <section>Fields</section>
-        </sections>
-    </layouts>
+  <fields>
+    <help>L'adreça actual que omplen els camps de facturació del compte.</help>
+    <label>Adreça actual</label>
+    <name>Current_Address__c</name>
+    <relationshipLabel>Comptes (adreça actual)</relationshipLabel>
+  </fields>
+  <fields>
+    <label>_SYSTEM: One2OneContact</label>
+    <name>Primary_Contact__c</name>
+    <relationshipLabel>Compte</relationshipLabel>
+  </fields>
+  <layouts>
+    <layout>HEDA Organization Layout</layout>
```

Comment:
@ceiroa @GloryChoe HEDA Organization block should be ordered after the HEDA Household Block. I know it seems nit picky but this means that it will likely show as a conflict and then a dev has to deploy to a scratch org, pull it back down (because the platform will change the order for us) and then merge from that branch. It's an extra step.